### PR TITLE
Parse user from URL for dashboard.html (useful for sharing)

### PR DESCRIPTION
**Pull request by azat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-2** [comment]
*This is an automated comment for commit 02bcb6a825ca951560b5f2f4799666c3bf21e7a0 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/65556/02bcb6a825ca951560b5f2f4799666c3bf21e7a0/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Cloud fork sync (only for ClickHouse Inc. employees)</td><td>If it fails, ask a maintainer for help</td><td>⏳ pending</td></tr>
<tbody>
</table>
<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>Checks correctness of the PR's body</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: programs/server/dashboard.html [hunk 1]
```diff
@@ -1077,7 +1085,7 @@
     user = document.getElementById('user').value;
     password = document.getElementById('password').value;
 
-    let url = `${host}?default_format=JSONColumnsWithMetadata&enable_http_compression=1`
```
Comment (alexey-milovidov) [inline programs/server/dashboard.html:1080]: Why not?

Comment (azat) [reply programs/server/dashboard.html:1080]: Egh, sorry, this changed accidentally, reverted.

**Comment by alexey-milovidov** [comment]
@azat I fixed a test that was failing here. Please merge with the master branch.

**Comment by azat** [comment]
And now CI completely hangs - https://github.com/ClickHouse/ClickHouse/actions/runs/9755347440

>An unexpected error has occurred and we've been automatically notified. Errors are sometimes temporary, so please try again.
>
>If the problem persists, please check whether the Actions service is operating normally at https://githubstatus.com. If not, please try again once the outage has been resolved.
>
>Should you need to contact Support, please visit https://support.github.com/contact and include request ID: 7218a65ba01ad29bd805ea43dae8ddfd

Will do one more rebase (sigh)

**Comment by azat** [comment]
Too much for a change that even don't have tests

**Comment by alexey-milovidov** [comment]
No worries. We are aiming to fix all flaky tests. It means, we can use arbitrary PRs with single-line changes to additionally validate all our tests.

**Pull request merged**